packages/account/src/Containers/Account/page-overlay-wrapper.tsx

```diff
@@ -23,11 +24,11 @@ type PageOverlayWrapperProps = {
  */
 const PageOverlayWrapper = observer(({ routes, subroutes }: PageOverlayWrapperProps) => {
     const history = useHistory();
-    const { client, common, ui } = useStore();
-    const { is_mobile } = ui;
+    const { client, common } = useStore();
     const { logout } = client;
     const { is_from_derivgo } = common;
     const { is_next_wallet_enabled } = useFeatureFlags();
+    const { isDesktop } = useDevice();
```

Comment:
Why are we using isDesktop and negating it? in Ln 62? Can we use isMobile? Is it bnecause the same feature is required in mobile and tablet?

Reply:
instead of this we can add `isMobile && isTablet && selected_route` but this will become huge condition. 
By adding `!isDesktop` even if in future we will need to add more breakpoint like `isSmallMobile` or any other breakpoint we will not have to add any breakpoint here in Ln 62 as well.

Yes its because we need same feature in mobile and tablet. 🙇